### DISCUSSION TITLE
책빵 : 책 구절을 쉽게 보는 앱

### DISCUSSION THREAD

**Discussion by morningfest** [DeveloperAcademy-POSTECH/.github#63] [Show and tell]
안녕하세요 저는 아카데미 3기 김리입니다.
이 앱은 저희 MC3 애플퍼프걸스가 열심히 만든 앱입니다.

https://github.com/user-attachments/assets/e2962002-5a0b-447b-b5f2-93d54b99f3cc

이런 앱입니다.
책별 혹은 클립(카테고리)별로 구절들을 볼 수 있는 앱입니다.

아직은 미완성이지만, 완성해서 배포하는 것이 목표입니다!

**Comment by 3405000** [comment]
멋져요

**Comment by yeniful** [comment]
우와 책 읽을 때 기록하는걸 너무 좋아하고, 중요하게 생각하는 1인인데
출시하면 꼭 말씀해주세요 !!!! 써보고싶어요!!!! 화이팅 🍀🍀
